Remove progress blockers from iOS docs. (#766)
spatialos/gdk-for-unity: docs/content/mobile/ios/local-deploy.md

## iOS Simulator{#ios-simulator}

1. Open your project in the Unity Editor.

> 1. In the Unity Editor, select **File** > **Build Settings**, and ensure that **iOS** is selected. Selection is indicated by a Unity logo that appears next to the name of the selected platform.<br>

Reviewer: ```suggestion
1. In your Unity Editor, select **File** > **Build Settings**, and ensure that **iOS** is selected. (The Unity logo next to a platform name indicates it's selected.)<br>
```

If **iOS** is not selected, select it and then select **Switch Platform**.
1. In the Unity Editor, select **Edit** > **Project Settings** > **Player**. This opens **PlayerSettings** in the Inspector window.
1. In the Inspector window, select **Settings for iOS (the iPhone icon)** > **Other Settings**.
1. In the **Configuration** section of the Inspector window, locate **Target SDK** and select **Simulator SDK**.
1. Still in the **Configuration** section, locate **Target minimum iOS version** and input `10.0`.
1. Build your workers from the SpatialOS menu by selecting **Build for local** > **All workers**.
1. In the Unity Editor, select **SpatialOS** > **Local launch**.<br>
It’s done when you see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`.

> 1. In Finder, navigate to `/workers/unity/build/worker/` and locate the `.xcodeproj` that corresponds to your iOS client-worker, it may be in a sub-folder.<br>

Reviewer: Same suggestion as above.

	Open it in Xcode.
1. Still in XCode, select the **Play** button in the top left of the window.
1. Once the game is deployed and started on the Simulator, you see an empty text field and a **Connect** button: Select **Connect**.<br>
Note: You don’t need to enter anything in the text field.
1. Play the game on the Simulator.

> 1. When you're done, exit the Simulator.<br>

Reviewer: Same suggestion as above.

	In XCode, select the **Play** button to stop your the game running.<br>
	In the terminal window that's running the SpatialOS process, enter **Ctrl+C** to stop the process.

## iOS device{#ios-device}

1. Make sure your computer and your mobile device are both connected to the same network.
1. Connect the mobile device to your computer using a USB cable.
1. Open your project in the Unity Editor.

> 1. In the Unity Editor, select **File** > **Build Settings**, and ensure that **iOS** is selected. Selection is indicated by a Unity logo that appears next to the name of the selected platform.<br>

Reviewer: Same suggestion as above.

1. In the Inspector window, select **Settings for iOS (the iPhone icon)** > **Other Settings**.
1. In the **Configuration** section of the Inspector window, locate **Target SDK** and select **Simulator SDK**.
1. Still in the **Configuration** section, locate **Target minimum iOS version** and input `10.0`.
1. Build your workers from the SpatialOS menu by selecting **Build for local** > **All workers**.
1. You need to know the local IP address of your computer to connect. [This page](https://lifehacker.com/5833108/how-to-find-your-local-and-external-ip-address) (on the Lifehacker website)  describes how you can find your external and local IP address.
1. In a terminal window from the root folder of your SpatialOS project,  run: `spatial local launch --runtime_ip=<your-local-ip>`. (Where `<your-local-ip>` is the IP address you just located.)<br>
You cannot use **SpatialOS** > **Local launch** in your Unity Editor as you would normally, because you need to specify the runtime IP.<br>
It’s done when you see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`.

> 1. In Finder, navigate to `/workers/unity/build/worker/` and locate the `.xcodeproj` that corresponds to your iOS client-worker, it may be in a sub-folder.<br>

Reviewer: Same same.
